x-pack/plugins/security_solution/public/common/components/matrix_histogram/index.tsx

```diff
@@ -181,8 +181,10 @@ export const MatrixHistogramComponent: React.FC<MatrixHistogramComponentProps> =
     useMatrixHistogramCombined(matrixHistogramRequest);
   const [{ pageName }] = useRouteSpy();
 
-  const onHostOrNetworkPage =
-    pageName === SecurityPageName.hosts || pageName === SecurityPageName.network;
+  const onHostOrNetworkOrUserPage =
```

Comment:
Hey @angorayc!

Do you know why we only show the new icon on these pages?

Reply:
That's a decision we made that visualization actions are only available in Explore section.

Reply:
As long as we have confirmed with Paul that we want visualization actions on user page, then all good